vault/core.go

```diff
@@ -2472,6 +2497,9 @@ func (c *Core) preSeal() error {
 	if err := c.unloadMounts(context.Background()); err != nil {
 		result = multierror.Append(result, fmt.Errorf("error unloading mounts: %w", err))
 	}
+	if err := c.teardownNamespaceStore(); err != nil {
```

Comment:
Does LoginMFA shutdown need to occur before NamespaceStore teardown? Can it reference namespaces?

@schultz-is thinks it might be able to.

Reply:
LoginMFA is handled by #1336.